Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java

```diff
@@ -506,9 +522,17 @@ public void setSelectedDocument(int selectedDocument) {
      */
     private List<SelectItem> populateDocumentTypes() {
         List<SelectItem> localDocumentTypeList = new ArrayList<SelectItem>();
-        localDocumentTypeList.add(new SelectItem("Prescription", "Prescription"));
-        localDocumentTypeList.add(new SelectItem("Discharge Summary", "Discharge Summary"));
-        localDocumentTypeList.add(new SelectItem("Transfer of care referral note", "Transfer of care referral note"));
+
+        try {
+            //Load the documentType.properties file
+            Properties localDocumentTypeProperties = PropertyAccessor.getInstance().getProperties(NhincConstants.DOCUMET_TYPE_PROPERTY_FILE);
+            for (Iterator<Entry<Object, Object>> it = localDocumentTypeProperties.entrySet().iterator(); it.hasNext();) {
```

Comment:
this should be a while loop. while(it.hasNext()).  Common variable name for iterator is iter or even itr.